UnitTests/resources/TestSolution/TestSolution.sln

```diff
@@ -0,0 +1,34 @@
+﻿

```

Comment:
Does the TestSolution serve any purpose now? It doesn't appear to be used by any test